exercises/Paul-Sobers/short-response.md

```diff
@@ -0,0 +1,15 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+Answer: Functions is created with the keyword function , followed by a function name and a set of () which contain the parameters and {}. Parameters are like regular variables except this variable is used as an input for the function when creating it like a function name. Inside the curly braces contain statements that are to be executed when the function is called. Arguments are real values passed to the function.
+
+2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments? 
+Answer: When you invoke a JavaScript function with a missing argument console returns a error telling you undefined. When you call a extra argument JavaScript ignores that argumentt and only returns the first argument.
+
+3. Define _hoisting_. What impact does hoisting have on the way that we structure our JavaScript programs?
+Answer: Hoisting is when you call your function before writing it. Meaning calling your function above your function code. The impact hoisting affects the way we structure our JavaScript programs because since only functions declaration can only be hoisted this let us only use let and const on one variable essentially making our code easy to read and write.
```

Comment:
This is not correct. Conceptually, you're on the right track but you have some work to do. Read [this](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) and use as much language from this article as possible in your response without directly copying and pasting.